applications/openshift/api-server/api_server_no_adm_ctrl_plugins_disabled/rule.yml

```diff
@@ -40,15 +45,15 @@ ocil: |-
 
 warnings:
 - general: |-
-    {{{ openshift_filtered_cluster_setting({'/api/v1/namespaces/openshift-kube-apiserver/configmaps/config': jqfilter}) | indent(4) }}}
+    {{{ openshift_filtered_cluster_setting({apipath: dump}) | indent(4) }}}
 
 
 template:
   name: yamlfile_value
   vars:
     ocp_data: "true"
     filepath: |-
-      {{{ openshift_filtered_path('/api/v1/namespaces/openshift-kube-apiserver/configmaps/config', jqfilter) }}}
+      {{{ openshift_filtered_path(apipathdefault, jqfilter) }}}
```

Comment:
@Vincent056  Could you tell me why you think `openshift_filtered_path(path, filter)` requires additional parameter? In my understanding, the return value of `openshift_filtered_path()` will be used by `oscap` command, and therefore only default (dump) path is required as I wrote here.

Reply:
Hi @tmishina Sorry for the confusion, it was meant to say `openshift_cluster_setting`